packages/calcite-components/src/components/stepper-item/stepper-item.scss

```diff
@@ -348,6 +335,44 @@
   }
 }
 
+:host([layout="horizontal"][single-view-mode]) .stepper-item-header {
+  @apply border-none 
+  me-0;
+  inline-size: 100%;
+  padding-inline: calc(var(--calcite-internal-stepper-action-inline-size) + 0.5rem);
+}
+
+:host([layout="horizontal"][single-view-mode][error]) .container {
+  & .stepper-item-number {
+    color: theme("backgroundColor.danger");
+  }
+  & .stepper-item-icon {
+    @apply opacity-100;
+    color: theme("backgroundColor.danger");
+  }
+}
+
+:host([layout="horizontal"][single-view-mode][error][selected]),
+:host([layout="horizontal"][single-view-mode][complete][selected]) .container {
+  @apply text-color-3;
+  & .stepper-item-heading {
+    @apply text-color-2;
+  }
+}
+
+:host([layout="horizontal"][single-view-mode][complete][selected]) .container {
+  & .stepper-item-icon {
+    @apply opacity-disabled;
+  }
+  & .stepper-item-number {
+    @apply text-color-3;
+  }
+}
+
+:host([layout="horizontal"][data-display="hide"]) {
```

Comment:
Is it possible to use `hidden` vs `data-display="hide"` instead? `data-display="visible"` doesn't seem to be used.